packages/connect/src/types/fees.ts

```diff
@@ -39,3 +39,5 @@ export const SelectFeeLevel = Type.Union([
         total: Type.String(),
     }),
 ]);
+
+export type FeeLevelLabel = FeeLevel['label'];
```

Comment:
probably not a big deal adding this, but lets keep in mind that with every new export from connect we have more complicated live in terms of keeping things backwards compatible

Reply:
I can move this to `suite-common` so the responsibility is on the side of the Suite. Would it be better for you?  `FeeLevel` is used there already anyway.

Reply:
I'd probably move it out of connect. not saying it can't be there, just that when it becomes part of public API, we should think twice whether it makes sense or not.